src/ASM/LR/ASMLRSpline.C

```diff
@@ -143,15 +143,40 @@ bool ASMunstruct::refine (const LR::RefineData& prm,
 {
   PROFILE2("ASMunstruct::refine()");
 
+  auto&& storeMesh = [this, fName]()
```

Comment:
Lambda functions is a nice concept, but in this case I think a protected member function would be a more natural choice?

Reply:
Probably, i just did the same as was done for mixed.